README.md

```diff
@@ -165,12 +165,22 @@ library
   default-language:    Haskell2010
 ```
 
-If you want to be able to import `Extra.*` modules when using `mixins` approach,
-you need to list those modules under `mixins` field as well, like this:
+This way, you can't access any other module of relude. If you want to use e.g.
+`Relude.Extra` (which reexports all `Relude.Extra.*` modules), you can add it
+like this:
```

Comment:
I would also leave the wording "you need to list those modules under `mixins` field as well, like this:" as it gives more understanding of what is happening below.